plugins/accuracy.cc

```diff
@@ -277,14 +285,31 @@ runCategorical(AccuracyConfig & runAccuracyConf,
                         outputRow.emplace_back(score + columnName, v, recordDate);
                     }
 
+                    if (accuracyOverN > 1) {
+                        bestlabelsCandidates.emplace_back(v, 
+                            CellValue(jsonDecodeStr<CellValue>(columnName.toSimpleName())));
```

Comment:
You don't need the CellValue constructor here.  Also, you should use CellValue(columnName), not pass through strings.